kroxylicious-runtime/src/main/java/io/kroxylicious/proxy/KafkaProxy.java

```diff
@@ -198,19 +207,15 @@ public void block() throws Exception {
 
     /**
      * Shuts down a running proxy.
-     * @throws InterruptedException
      */
-    public void shutdown() throws InterruptedException {
+    private void shutdown() {
```

Comment:
This isn't a backwards compatible change so I'm curious to hear the justifications.

Reply:
Is `KafkaProxy` a public API?

Reply:
I'm happy to close my eyes.

Reply:
Err yes. Its where any user embedding the proxy interacts with the proxy, no? Which also happens to be the target persona for the fix in question.